### PR TITLE
Feature | View Workplaces Permission UI

### PR DESCRIPTION
**Pull request by ChazUK**
There's one piece left on this that requires BE work, so going to merge this now and wait for the extra piece.

### CODE REVIEW

File: src/app/features/workplace/workplace-info-panel/workplace-info-panel.component.html [hunk 1]
```diff
@@ -13,36 +13,33 @@
...
-          </ng-container>
+          {{ workplace.dataOwner }}
         </p>
       </div>
-      <div class="govuk-grid-column-one-half">
-        <p class="govuk-!-margin-bottom-1"><strong>Data owner</strong></p>
+
+      <div class="govuk-grid-column-three-quarters">
+        <p class="govuk-!-margin-bottom-1">
+          <strong>{{ workplace.dataOwner | workplacePermissionsBearer }} view only permissions</strong>
```
Comment (omarCreativeDev) [inline src/app/features/workplace/workplace-info-panel/workplace-info-panel.component.html]: think you need the double ||

File: src/app/features/workplace/workplace-info-panel/workplace-info-panel.component.html [hunk 2]
```diff
@@ -13,36 +13,33 @@
...
+        </p>
         <p class="govuk-!-margin-bottom-0">
-          {{ workplace.dataOwner }}
-          <span class="govuk-util__float-right">Change data owner</span>
+          {{ workplace.dataPermissions | dataViewPermissions }}
```
Comment (omarCreativeDev) [inline src/app/features/workplace/workplace-info-panel/workplace-info-panel.component.html]: same here ||

File: src/app/features/workplace/workplace-info-panel/workplace-info-panel.component.html (hunk 1 above)
Comment (ChazUK) [reply src/app/features/workplace/workplace-info-panel/workplace-info-panel.component.html]: It's a pipe

File: src/app/features/workplace/workplace-info-panel/workplace-info-panel.component.html (hunk 2 above)
Comment (ChazUK) [reply src/app/features/workplace/workplace-info-panel/workplace-info-panel.component.html]: pipe!

**Comment by ChazUK** [comment]
[(2) Parents and Subs - Update View workplaces view](https://trello.com/c/EnNrPZAG/9-2-parents-and-subs-update-view-workplaces-view)

**Pull request merged**